src/components/Home.tsx

```diff
@@ -1,9 +1,19 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import { view } from 'react-easy-state';
+import Header from './Header';
+import SearchBar from './SearchBar'
 
 const Home = view(() => {
-    return <h1>Home!</h1>
+    return <>
+        <Header title='Safe NPM' grade='null'></Header>
```

Comment:
try not to pass a string value `"null"`, either use a valid value, or an actual `null` value